### PR TITLE
cli: `common.PrintVerbose` prints via `cobra.Command.Printf`

### LINKED ISSUE #1962: cli: `PrintVerbose` writes to stdout instead of command out
https://github.com/nspcc-dev/neofs-node/blob/41f93dcc1dea5001a5674c37847fdca4d6280ab1/cmd/neofs-cli/internal/common/verbose.go#L18
Is it expected? I think we should write all the data to commands' streams.

### PR DESCRIPTION
**Pull request by acid-ant**
Close #1962

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: cmd/neofs-cli/internal/client/sdk.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func GetSDKClient(key *ecdsa.PrivateKey, addr network.Address) (*client.Client,
 }
 
 // GetCurrentEpoch returns current epoch.
-func GetCurrentEpoch(ctx context.Context, endpoint string) (uint64, error) {
+func GetCurrentEpoch(cmd *cobra.Command, ctx context.Context, endpoint string) (uint64, error) {
```
Comment (roman-khimov) [inline cmd/neofs-cli/internal/client/sdk.go:72]: `revive` linter is not happy about this change.

Comment (acid-ant) [reply cmd/neofs-cli/internal/client/sdk.go]: Thanks, updated.

File: cmd/neofs-cli/modules/util/convert_eacl.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ func convertEACLTable(cmd *cobra.Command, _ []string) {
 func prettyPrintJSON(cmd *cobra.Command, data []byte) {
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/util/convert_eacl.go:58]: BTW, can this be replaced with `common.PrettyPrintJSON()`?

File: cmd/neofs-cli/modules/container/get.go [hunk 3]
```diff
@@ -79,12 +79,12 @@ func prettyPrintContainer(cmd *cobra.Command, cnr container.Container, jsonEncod
 	if jsonEncoding {
```
Comment (roman-khimov) [inline cmd/neofs-cli/modules/container/get.go:79]: And this one too?

**Review by roman-khimov — COMMENTED** [review]
On one hand it's OK since it solves the problem. On the other I'm think about changing `common` to `interactive` or `cmdctx` of some kind with `interactive.Command` (or similar) type that would alias `*cobra.Command` and implement `PrettyPrintJSON`, `PrintVerbose`, `ReadEACL` and all the other methods. Maybe it'd be easier to instantiate this new type once then use it instead of adding `cmd` everywhere, because it looks to me that having a number of print calls in a function (chain of functions) is a more popular scenario than a single occasional print.

Just a suggestion though.

**Comment by fyrchik** [comment]
@roman-khimov could you elaborate a bit? We cannot define methods on non-internal type _aliases_.
So do you mean something like this:
```
package interactive
type Command cobra.Command // and define all methods we need
type Command struct { // and define only additional methods
  *cobra.Command
}
```
and then refactoring all functions to make use of this new type?

**Comment by roman-khimov** [comment]
> type Command cobra.Command // and define all methods we need

More like this one (yes, technically it's not an alias, but a separate type). At least until we have some need for a real `struct`.

File: cmd/neofs-cli/modules/util/convert_eacl.go (hunk 2 above)
Comment (acid-ant) [reply cmd/neofs-cli/modules/util/convert_eacl.go]: Done.

File: cmd/neofs-cli/modules/container/get.go (hunk 3 above)
Comment (acid-ant) [reply cmd/neofs-cli/modules/container/get.go:77]: Yes, replaced.

**Review by carpawell — APPROVED** [review]
Solves the original issue, all the comments could be solved via a separate PR, IMO.

**Pull request merged**